.github/workflows/rust-CI.yml

```diff
@@ -0,0 +1,64 @@
+on: [push, pull_request]
+
+name: Rust Continuous integration
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  test:
+    name: Test Suite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: test
```

Comment:
@AntoineRR , are these the replacements of the `circle ci`?

These are `cargo test` and `cargo check` , right?

`cargo test` is already being done in the `.circleci/config.yml` file

Reply:
Thanks for your feedback :)

From what I understand of CircleCI, you indeed defined a `test` command in `.circleci/config.yml` on line 12, that would run `cargo test`. However, this command is currently not used in any of the workflows. To use the `test` command, it should be added under `steps` inside a `job` (example from the [doc](https://circleci.com/docs/reusing-config#invoking-reusable-commands): the command `sayhello` is invoked inside the `myjob` job, which has to be run by a workflow). As I am not very familiar with this, I may have misunderstood something, so feel free to correct me if that's the case.

I also chose to add the commands to github workflows as it is easier to specify the CI should run on push AND on pull requests. To move `cargo test`, `cargo fmt` and `cargo clippy` to circle CI, I think I would just have to uncomment lines 56-57 of `.circleci/config.yml` to invoke the `lint-test-build` job defined in the rust orb :) Let me know what suits you best!

Reply:
@AntoineRR , that makes sense. I anyway want to move away from CircleCI. This will be a good first step.

Thanks!